harness/determined/core/_tensorboard_mode.py

```diff
@@ -0,0 +1,21 @@
+import enum
+
+
+class TensorboardMode(enum.Enum):
+    """
+    ``TensorboardMode`` defines how Tensorboard artifacts are handled.
```

Comment:
tiniest nit: add an empty line after this line, so that there is a paragraph separation in the rendered docs.